### ISSUE TITLE
Accessibility: Use of Fieldset

### ISSUE THREAD

**Issue by cyokum** [Fastspot/bigtree-form-builder#15]
Single input fields should not have a fieldset tag. This includes: email, text, textarea, upload, and url.

Grouped fieldsets such as phone, address, and name are correctly using the fieldset tags and the legend tag.

**Comment by timbuckingham** [comment]
I think whatever tag was has gone missing in the GitHub filter :)

**Comment by cyokum** [comment]
Can you just guess? JK. Updated.
